main/display/display.c

```diff
@@ -9,6 +9,7 @@
 #include <esp_heap_caps.h>
 
 #include "display.h"
+#include "bresenham.h"
```

Comment:
Organizationally, I don't love `components` having non-`component` dependencies. If this is just for drawing purely horizontal lines, it seems like overkill. Write a tiny `for` loop!

Reply:
Oh yeah, that would make a lot of sense. I'll swap it out.

Reply:
Done